app/scripts/modules/core/src/pipeline/config/triggers/artifacts/triggerArtifactConstraintSelector.less

```diff
@@ -12,7 +12,7 @@
 }
 
 .artifact-select .Select-arrow-zone {
-  border-left: 1px solid rgb(204, 204, 204);
+  border-left: 1px solid var(--color-alto);
```

Comment:
@sbwsg This gets rid of the lint warning. Hope this ok. (They are the same color)